__tests__/app.test.js

```diff
@@ -138,3 +137,46 @@ describe("/api/users", () => {
     });
   });
 });
+describe("/api/reviews", () => {
+  describe("GET", () => {
+    test("200: should return an array of objects, each with properties owner, title, review_id, category, review_img_url, created_at, votes, designer, comment_count. reviews should be sorted by date in desc order", () => {
+      return request(app)
+        .get("/api/reviews")
+        .expect(200)
+        .then(({ body }) => {
+          expect(body.reviews).toBeInstanceOf(Array);
+          expect(body.reviews).not.toHaveLength(0);
```

Comment:
we can be more exact with the length since we know how many reviews are in our test database